Src/VimCore/EditorUtil.fs

```diff
@@ -1818,13 +1818,19 @@ module TextViewUtil =
     /// Be aware when using GetTextViewLineContainingYCoordinate, may need to add the
     /// _textView.ViewportTop to the y coordinate
     let GetTextViewLines (textView: ITextView) =
-        try
-            let textViewLines = textView.TextViewLines
-            if textViewLines <> null then Some textViewLines
-            else None
-        with 
+        if textView.IsClosed || textView.InLayout then
+
             // TextViewLines can throw if the view is being laid out.  Highly unlikely we'd hit
             // that inside of Vim but need to be careful

```

Comment:
It's actually likely during startup that we end up executing VsVim commands during a layout. Over the years I've had a number of bug reports in this area. Essentially every startup sequence for different file types is a bit of a race condition. As such the editor ends up firing off events while it's still in the middle of a layout. Once we're past startup though it's pretty unlikely.

Reply:
Unfortunately I can think of no easy way to force F# clients to use our bulletproof API. At least when trying to do something in VsVim, people like me who search around for something similar to build off of will now only find uses of the preferred API.